### ISSUE TITLE
[🐛 Bug]: starter project generated with @wdio/cli does not run

### ISSUE THREAD

**Issue by Badisi** [webdriverio/create-wdio#14]
### WebdriverIO Version

7.16.3

### Node.js Version

14.17.5

### Mode

Standalone Mode

### What happened?

The starter project generated with `@wdio/cli` does not run.

Following the [getting started guide](https://webdriver.io/docs/gettingstarted):
1) Initiate a setup
```sh
npx wdio .
```
<img width="627" alt="Screenshot 2021-11-02 at 11 26 24" src="https://user-images.githubusercontent.com/674862/139829730-5b20e686-8009-4db9-9b99-dfdf3fe11889.png">

2) Run the test
```sh
npx wdio run ./wdio.conf.js
````

This outputs the following error:
> Error: Could not execute "run" due to missing configuration. Would you like to create one? (y/N) 

### Issue

The issue here is that the wizard created a `wdio.conf.ts` and not a `wdio.conf.js` as stated in the doc (or as configured in the package.json script) 

### Attempt 1

Replacing the command with:
```sh
npx wdio run ./wdio.conf.ts
```
gives the following issue:

> SyntaxError: Unexpected token 'export'

### Workaround

Renaming `wdio.conf.ts` to `wdio.conf.js` and editing the file as follow:
```js
// replace:
export const config: WebdriverIO.Config = {

// with:
exports.config = {
```

### What is your expected behavior?

I expect the starter project to run without issues.

### How to reproduce the bug.

Simply follow the [getting started guide](https://webdriver.io/docs/gettingstarted):
1) Initiate a setup
```sh
npx wdio .
```
2) Run the test
```sh
npx wdio run ./wdio.conf.js
```

### Relevant log output

```typescript
export const config: WebdriverIO.Config = {
^^^^^^

SyntaxError: Unexpected token 'export'
    at wrapSafe (internal/modules/cjs/loader.js:988:16)
    at Module._compile (internal/modules/cjs/loader.js:1036:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1101:10)
    at Module.load (internal/modules/cjs/loader.js:937:32)
    at Function.Module._load (internal/modules/cjs/loader.js:778:12)
    at Module.require (internal/modules/cjs/loader.js:961:19)
    at require (internal/modules/cjs/helpers.js:92:18)
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

**Comment by christian-bromann** [comment]
The problem is here: https://github.com/webdriverio/wdio/blob/main/src/index.ts#L105 .. we add a script to the package.json without checking whether TypeScript was chosen or not. I suggest to add a check and name the config file correctly. Contributions are appreciated.

**Comment by Badisi** [comment]
@praveendvd, I was trying to configure `webdriverio` in an existing `angular` project which already had a `tsconfig.json` file, so you were right about that, my existing ts config was not made for `wdio`.

So what I did was creating a new `tsconfig.json` file in the `./test` folder and specify it during the run command.

```sh
wdio run wdio.conf.ts --autoCompileOpts.tsNodeOpts.project=test/tsconfig.json
```

#### test/tsconfig.json
```json
{
    "compilerOptions": {
        "types": [
            "node",
            "@wdio/sync",
            "@wdio/jasmine-framework",
            "expect-webdriverio",
            "@wdio/devtools-service",
            "@wdio/appium-service"
        ],
        "target": "es5"
    }
}
```

@christian-bromann is also right as my `package.json` add the following script :
```json
"scripts": {
        "wdio": "wdio run wdio.conf.js"
}
````
even though I had chosen `typescript` as a compiler.

**Comment by praveendvd** [comment]
> The problem is here: https://github.com/webdriverio/wdio/blob/main/src/index.ts#L105 .. we add a script to the package.json without checking whether TypeScript was chosen or not. I suggest to add a check and name the config file correctly. Contributions are appreciated.

I create a package.json first using `npm init -y` then install `npm install @wdio/cli` and does the default project creationg 

For me there is no script tag created as you mentioned

**Comment by Badisi** [comment]
My test case:

```sh
$ mkdir test
$ cd test
$ npx wdio .
```

Using the following answers:
![image](https://user-images.githubusercontent.com/674862/139873278-86a4a10d-ffc0-4c93-a2db-4eb87841f46c.png)

Result:
<img width="450" alt="Screenshot 2021-11-02 at 16 09 09" src="https://user-images.githubusercontent.com/674862/139874441-e9a59be9-94c8-46e7-9af5-c7c0f553b635.png">

It says that I can use either `npx wdio run wdio.conf.ts` or `npm run wdio`.

And in fact, for the latter, the `package.json` generated is:
<img width="427" alt="Screenshot 2021-11-02 at 16 06 46" src="https://user-images.githubusercontent.com/674862/139874002-3243b7af-77e2-4735-9c29-c0b0ffd56e6a.png">

**Comment by praveendvd** [comment]
@Badisi  perfect thank you

**Comment by praveendvd** [comment]
@Badisi  Please feel free to contribute

**Comment by Badisi** [comment]
Fixed by https://github.com/webdriverio/create-wdio/pull/15

**Comment by JoostRituals** [comment]
I googled this, and ended up here.  TLDR: This is, as of this day and most recent version, still a valid bug.

With the latest version of WDIO, Latest LTS version of NodeJS, and when choosing TypeScript + the Test files, It's not working. The 'out of the box experience' one is trying to achieve with the Configuration Helper is not giving the expected result. 

This is the console output:
`export const config: WebdriverIO.Config = {^^^^^^`

This is a somewhat working 'workaround':
` 
// replace:
export const config: WebdriverIO.Config = {

// with:
exports.config = {
`

**Comment by Badisi** [comment]
@JoostRituals, this seems related to **`# Issue 1`** (here: [#306](https://github.com/webdriverio/create-wdio/issues/306)) and has been fixed in `create-wdio@8.2.0` published 2 days ago. 

I said "this seems" because, well, you are not giving any sufficient info about the error you are facing.
Try to be more precise next time ;-)

**Comment by JoostRituals** [comment]
@Badisi 
Thnx for the swift reply. I will be more precise next time. In this case, I had a carbon copy of your 'issue'.
